spine-ts/spine-pixi/src/Spine.ts

```diff
@@ -0,0 +1,407 @@
+import type { BlendMode, Bone, Event, NumberArrayLike, SkeletonData, Slot, TextureAtlas, TrackEntry } from "@esotericsoftware/spine-core";
+import {
+	AnimationState,
+	AnimationStateData,
+	AtlasAttachmentLoader,
+	ClippingAttachment,
+	Color,
+	MeshAttachment,
+	RegionAttachment,
+	Skeleton,
+	SkeletonBinary,
+	SkeletonClipping,
+	SkeletonJson,
+	Utils,
+	Vector2,
+} from "@esotericsoftware/spine-core";
+import type { SpineTexture } from "./SpineTexture";
+import { SlotMesh } from "./SlotMesh";
+import type { ISpineDebugRenderer } from "./SpineDebugRenderer";
+import { Assets } from "@pixi/assets";
+import type { IPointData } from "@pixi/core";
+import { Ticker, utils } from "@pixi/core";
+import type { IDestroyOptions, DisplayObject } from "@pixi/display";
+import { Container } from "@pixi/display";
+
+export interface ISpineOptions {
+	removeUnusedSlots?: boolean;
```

Comment:
What happens with slots that are inside skins? 🤔 

In any case, that was an idea that lead nowhere, the implementation now is not good since it removes the children one at a time. Since children is an array removing one at a time could lead to many splicings and that not the nicest way to empty an array...

I think we should remove that 😅